src/Services/Search/FhirSearchWhereClauseBuilder.php

```diff
@@ -34,7 +34,12 @@ public static function build($search, $isAndCondition = true): SearchQueryFragme
             foreach ($search as $key => $field) {
                 if (!$field instanceof ISearchField) {
                     // developer logic error
-                    throw new \BadMethodCallException("Method called with invalid parameter.  Expected SearchField object for parameter '" . $key . "'");
+                    // treat the field as an exact string match if they send us a primitive
+                    if (is_string($field) || is_numeric($field)) {
+                        $field = new StringSearchField($key, $field, SearchModifier::EXACT);
+                    } else {
+                        throw new \BadMethodCallException("Method called with invalid parameter.  Expected SearchField object for parameter '" . $key . "'");
+                    }
```

Comment:
Expected SearchField object for parameter

In this case an object is not expected so dev may start scratching their head.